client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -19,7 +19,7 @@
       {{/if}}
     </Ui::Legend>
 
-    {{#if (or this.hasModifiedZrSectionNumberQuestion this.hasZrSectionTitleQuestion)}}
+    {{#if (or this.hasModifiedZrSectionNumberQuestion this.hasZrSectionTitleQuestion (eq this.zrTypeLabel 'ZR'))}}
```

Comment:
Could this be made into a nicely-named computed property in the component class? That way we know what the logic here is trying to achieve in friendlier terms.

Reply:
Sounds good